src/main/java/qna/domain/Question.java

```diff
@@ -35,10 +42,26 @@ public Question(String title, String contents, User writer) {
         this.title = title;
         this.contents = contents;
         this.writer = writer;
+        this.answers = new Answers();
+    }
+
+    public void addAnswer(Answer answer) {
+        answers.add(answer);
     }
 
-    public boolean isOwner(User writer) {
-        return this.writer.equals(writer);
+    public void delete(User deleter) throws CannotDeleteException {
```

Comment:
```suggestion
    public List<DeleteHistory> delete(User deleter) {
```

1. checked exception을 unchecked exception으로 변경하세요.
2. 리턴타입으로 `List<DeleteHistory>`으로 바꾸고 DeleteHistories를 제거해보세요.
3. 리팩터링 하면서 메서드가 10줄이 넘어가지 않도록 만들어보세요.